### PR TITLE
Add wildfly.feature-pack.location parameter

### PR DESCRIPTION
**Pull request by tommaso-borgato**
## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [ ] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [ ] Code contributed by me aligns with current project license: [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)
 - [ ] Pull Request contains a description of the changes
 - [ ] Pull Request does not include fixes for multiple issues/topics
 - [ ] Code is self-descriptive and/or documented
 - [ ] I have implemented unit tests to cover my changes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

File: testsuite/src/test/java/org/jboss/intersmash/testsuite/provision/openshift/OpenShiftProvisionerTestBase.java [hunk 1]
```diff
@@ -143,6 +143,13 @@ public String eeFeaturePackLocation() {
 				return IntersmashConfig.getWildflyEeFeaturePackLocation();
 			}
 
+			@Override
+			public String featurePackLocation() {
+				// this value is supposed to be overridden by the CI Jenkins job by passing e.g.
```
Comment (fabiobrz) [inline testsuite/src/test/java/org/jboss/intersmash/testsuite/provision/openshift/OpenShiftProvisionerTestBase.java]: Could we please change this comment (here and in other similar occurrences) and avoid mentioning details?
E.g.: I'd do: `s/overridden by the CI Jenkins job/overridden externally/`

**Review by fabiobrz — APPROVED** [review]
Hi @tommaso-borgato and thanks for this PR. 
Changes LGTM but I've dropped one comment about one comment :smile: 
Can you please se whether you can fix? I don't wanna block though, so feel free to add an issue to track this work if you don't have time now.

File: testsuite/src/test/java/org/jboss/intersmash/testsuite/provision/openshift/OpenShiftProvisionerTestBase.java (hunk 1 above)
Comment (tommaso-borgato) [reply testsuite/src/test/java/org/jboss/intersmash/testsuite/provision/openshift/OpenShiftProvisionerTestBase.java]: @fabiobrz  done

**Comment by fabiobrz** [comment]
Thanks @tommaso-borgato - LGTM, merging!

**Pull request merged**